### PR TITLE
Implemented [Long pressed name](https://leetcode.com/problems/long-pressed-name/description/)

### PR DESCRIPTION
**Pull request by Sofbog**
taken from: https://leetcode.com/problems/long-pressed-name/description/
@Abnormaltype

### CODE REVIEW

File: task_list/python_long_pressed_name/long_pressed_name.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def is_long_pressed_name(self, name: str, typed: str) -> bool:
```
Comment (Abnormaltype) [inline task_list/python_long_pressed_name/long_pressed_name.py:1]: do you need `self` here?

File: task_list/python_long_pressed_name/long_pressed_name_test.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+from long_pressed_name import is_long_pressed_name
+
+
+def test_basic_true_cases():
+    """Test basic cases where the typed name is a valid long-pressed version of the name."""
```
Comment (Abnormaltype) [inline task_list/python_long_pressed_name/long_pressed_name_test.py:5]: comment in the prev PR

File: task_list/python_long_pressed_name/reference/long_pressed_name.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+        return False
+
+    i = 0
+    for j in range(len(typed)):
+        if i < len(name) and name[i] == typed[j]:
+            i += 1
+        elif j == 0 or typed[j] != typed[j - 1]:
+            return False
+
+    return i == len(name)
```
Comment (Abnormaltype) [inline task_list/python_long_pressed_name/reference/long_pressed_name.py:12]: do not use `i` and `j` as names for vars, `flake8` will mark it as errors

Comment (Sofbog) [reply task_list/python_long_pressed_name/reference/long_pressed_name.py:12]: got it,  flake8 didn’t return the error locally

Comment (Abnormaltype) [reply task_list/python_long_pressed_name/reference/long_pressed_name.py:12]: cuz you don't have all the needed plugins

Comment (Abnormaltype) [reply task_list/python_long_pressed_name/reference/long_pressed_name.py:12]: but if even yes, let's avoid this naming

File: task_list/python_long_pressed_name/python_long_pressed_name.en.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+Your friend is typing his `name` into a keyboard. Sometimes, when typing a character `c`, the key might get long pressed, and the character will be typed 1 or more times.
```
Comment (IvanRamyk) [inline task_list/python_long_pressed_name/python_long_pressed_name.en.md:1]:
```suggestion
Your friend is typing his `name` into a keyboard. Sometimes, when typing a character, the key might get long pressed, and the character will be typed 1 or more times.
```

Firstly, I though that only `c` could be doubled so I believe it's better to remove it. (and fix in all translations)

**Review by Abnormaltype — COMMENTED** [review]
your tests do not pass your proposed solution, please run them before pushing to check it.

**Review by Abnormaltype — COMMENTED** [review]
only one test covers `False` case. Why did you delete them?

File: task_list/python_long_pressed_name/long_pressed_name_test.py [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+def test_basic_false_cases():
+    assert not is_long_pressed_name(
+        "saeed", "ssaaedd"
+    ), "False: 'e' do not match."
+    assert not is_long_pressed_name("name", "nxaame"), "False: 'x' is not in 'name'."
+    assert not is_long_pressed_name("hello", "heoll"), "False: Incorrect order of characters."
+    assert not is_long_pressed_name("hello", "heo"), "False: Missing 'l' in 'typed'."
+    assert not is_long_pressed_name("start", "bstart"), "False: Extra character 'b' at the start."
+    assert not is_long_pressed_name("saeed", "ssaaedd"), "False: 'e' must have been pressed twice, but it was not in the typed output."
```
Comment (Abnormaltype) [inline task_list/python_long_pressed_name/long_pressed_name_test.py:30]: so, 
```
assert not is_long_pressed_name(
        "saeed", "ssaaedd"
    ), "False: 'e' do not match."
```
or 
```
assert not is_long_pressed_name("saeed", "ssaaedd"), "False: 'e' must have been pressed twice, but it was not in the typed output."
```
noo need to duplicate it

**Pull request merged**